metadata.json

```diff
@@ -67,13 +67,6 @@
     },
     {
       "operatingsystem": "Archlinux"
-    },
-    {
-      "operatingsystem": "AIX",
-      "operatingsystemrelease": [
-        "7.2",
-        "7.3"
-      ]
```

Comment:
I added AIX in a previous PR, but dropped Solaris...

Reply:
Wait, if we remove the default from `params.pp` because the same value is already in the main class:
* This is not a backwards-incompatble change;
* There is no reason to drop support of OS.

I do not use these systems nor this module, but if I did I would consider this removal as an intention from the maintainers to not maintain compatibility with these OS anymore.  If the OS is said to be supported, any failure would be considered a bug and would possibly be fixed (e.g. by providing fixed default values in Hiera).

Reply:
@smortex  AIX and Solaris has never been mentioned as supported in any release of this module.
see https://forge.puppetlabs.com/modules/pcfens/ca_cert/

Reply:
Changed the PR title a bit.
About backwards-incompatible i added the label in case anyone uses those defaults.

Reply:
Added an example for custom certificates handling